File: App/app/src/main/java/com/cinema/ticket_booking/data/model/request/RegisterRequest.java
package com.cinema.ticket_booking.data.model.request;

public class RegisterRequest {
    private String email, password, fullName, phone;

    public RegisterRequest(String email, String password, String fullName) {
        this.email = email;
        this.password = password;
        this.fullName = fullName;
        this.phone = null;
    }
}

File: App/app/src/main/java/com/cinema/ticket_booking/data/model/request/RegisterRequest.kt
package com.cinema.ticket_booking.data.model.request

import com.google.gson.annotations.SerializedName

data class RegisterRequest @JvmOverloads constructor(
    @SerializedName("email")
    val email: String?,
    @SerializedName("password")
    val password: String?,
    @SerializedName("fullName")
    val fullName: String?,
    @SerializedName("phone")
    val phone: String? = null
)
